manifests/s/Surfshark/Surfshark/5.7.1999/Surfshark.Surfshark.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Automatically updated by the winget bot at 2024/Apr/19
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Surfshark.Surfshark
+PackageVersion: 5.7.1999
+InstallerType: exe
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.surfshark.com/windows/latest/SurfsharkSetup.exe
+  InstallerSha256: 77A7BD0F9582C28CACD356BDD8C9159AC19E88B164010C54B181D32D60C3277B

```

Comment:
From the pipeline check : 
4/19/2024 4:10:51 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   44F4CC620042FFDA80CDDBDEB055193B6B416E923ECC0CA5406B6F93E855C742
Expected hash: 77a7bd0f9582c28cacd356bdd8c9159ac19e88b164010c54b181d32d60c3277b

4/19/2024 4:10:53 PM